src/NuGet.Clients/NuGet.VisualStudio.Client/NuGet.VisualStudio.Client.csproj

```diff
@@ -66,7 +66,7 @@
       <Link>Modules\NuGet\NuGet.PackageManagement.PowerShellCmdlets.dll-Help.xml</Link>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
-    <Content Include="$(SolutionPackagesFolder)newtonsoft.json\9.0.1\lib\net45\Newtonsoft.Json.dll">
+    <Content Include="$(SolutionPackagesFolder)newtonsoft.json\13.0.1\lib\net45\Newtonsoft.Json.dll">
```

Comment:
nit: You can use the newtonsoft version variable.

Reply:
The MSBuild property with the version number is defined in `packages.targets` which is imported after this line. I don't feel like spending time moving imports around to get this working without causing problems.

Reply:
Content is an item, but the version number is a property.
Property pass happens before the items pass, so the ordering wouldn't matter.

Reply:
Are you sure properties are evaluated before items?  I thought MSBuild was strictly "top down"

Reply:
Yep, https://docs.microsoft.com/en-us/visualstudio/msbuild/build-process-overview?view=vs-2019

Reply:
I'll try to remember for the cleanup issue.

Reply:
Only do that if you want to. 
I don't want these comments to add any additional work. 
It's nice to have, but really not that important in the grand scheme.